app/models/Service.scala

```diff
@@ -51,6 +79,10 @@ object Service {
 
   private val keyCacheRequest = "cacheServiceRequest-"
 
+  def clearCache() {
+    Cache.remove(keyCacheRequest)
+  }
```

Comment:
This is only working here because `clearCache` is of type `Unit` but if you wanted it to return a value then it wouldn't because you forgot the `=`symbol:

```scala
scala> def unit() {
     | "toto"
     | }
<console>:12: warning: a pure expression does nothing in statement position; you may be omitting necessary parentheses
       "toto"
       ^
unit: ()Unit

scala> def string() = {
     | "toto"
     | }
string: ()String
```

You should simply write:

```scala
def clearCache() = Cache.remove(keyCacheRequest)
```